vercel/public/lobby.css

```
@@ -0,0 +1,634 @@
+body {
+    position: static;
+    display: block;
+    background-color: #1a1a1a;
+    color: white;
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+    font-size: 100%;
+    font-family: Lucida Grande, Lucida Sans, Arial, sans-serif;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: Lucida

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)